streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -806,6 +803,17 @@ private KafkaStreams(final InternalTopologyBuilder internalTopologyBuilder,
         rocksDBMetricsRecordingService = maybeCreateRocksDBMetricsRecordingService(clientId, config);
     }
 
+    private long getCacheSizePerThread(final int numStreamThreads) {
+        return totalCacheSize / (numStreamThreads + ((globalTaskTopology != null) ? 1 : 0));
+    }
```

Comment:
IMO, the code would be easier navigable if you inline this method. Without the removed check, there is not really a reason to have a method here.

Reply:
I think I agree with you. fixed